ci/jenkins/test.sh

```diff
@@ -50,10 +50,9 @@ NETWORKPOLICY_SKIP="should allow egress access to server in CIDR block|should en
 CONTROL_PLANE_NODE_ROLE="master|control-plane"
 
 CLEAN_STALE_IMAGES="docker system prune --force --all --filter until=4h"
-CLEAN_STALE_IMAGES_CONTAINERD="crictl rmi --prune"
-
+CLEAN_STALE_IMAGES_CONTAINERD="sudo crictl rmi --prune"
```

Comment:
Did you encounter any errors when running the test without using sudo previously?

Reply:
Yes I got an error show that user Jenkins cannot access /var/run/containerd.sock because permission denied.  So I added `sudo`, it works well then.

Reply:
+1, I think it work well before without sudo, it maybe caused by incorrect user permission in the jumper node?

Reply:
Thanks, I'll double check the configuration on the jumper.

Reply:
After verification, I found that the Jenkins user do cannot access `/var/run/containerd.sock`.  After discussion with @XinShuYang , we determine to add permission when deploying the testbed. So `sudo` can be removed.